NN_chatCmd.js

```diff
@@ -12,17 +13,28 @@ nerthus.chatCmd.run = function(ch)
         var callback = this.fetch_callback(cmd,ch)
         if(callback)
         {
-            log("["+ch.k+"] " + ch.n + " -> " + ch.t) //gdze kto co
-            return callback(ch)
+            log("["+ch.k+"] " + ch.n + " -> " + ch.t +
+            " (" + nerthus.chatCmd.current_map +")"); //gdzie kto co mapa
+            return callback(ch);
         }
     }
 }
 
 nerthus.chatCmd.fetch_cmd = function(ch)
 {
-    if(ch.t[0]=='*')
-        return RegExp(/^\*(\S+)/).exec(ch.t)
-    return null
+    if (nerthus.chatCmd.currentMap === '')
+        nerthus.chatCmd.currentMap = map.name.replace(/\s/g, '');
+    if (ch.s === 'entertown')
+    {
```

Comment:
Nie wydaje mi się aby to było dobre miejsce na to. Prędzej widziałbym to jako a'la komendę 'entertown' czy co tam można wyciągnąć z komunikatu na chacie i w niej umieścił to banglanie widocznością.